socorro/tests/external/es/test_connection_context.py

```diff
@@ -2,62 +2,62 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import datetime
-
+from socorro import settings
+from socorro.libclass import build_instance
 from socorro.lib.libdatetime import utc_now
 
 
 class TestConnectionContext:
-    def test_create_index(self, es_conn):
-        # Delete any existing indices first
-        for index in es_conn.get_indices():
-            es_conn.delete_index(index)
-        es_conn.refresh()
-        es_conn.health_check()
+    def build_conn(self):
+        return build_instance(
+            class_path="socorro.external.es.connection_context.ConnectionContext",
+            kwargs=settings.CRASH_DESTINATIONS["es"]["options"],
```

Comment:
One of the things I spent a lot of time thinking about was how to test everything. Should we be testing with the mozilla_settings? Should we create a second set of settings for testing? The latter is less fragile to how we change mozilla settings around over time. But the former makes it more likely that "if it passes in CI, it'll work in prod".

If we had more time / resources, we could have test_settings AND also test out the mozilla_settings. The latter probably done with integration tests.

However, I don't have time / resources to maintain all that, so for now, we're testing with the mozilla_settings with some minor adjustments (bucket names, index names) so that test settings don't stomp on local dev environment crash storage.